spec/controllers/account/workspaces_controller_spec.rb

```diff
@@ -30,10 +30,12 @@
       get :show, params: { id: workspace.id }
       expect(response).to be_successful
     end
+  end
 
-    it 'should show pdf page for workspace' do
-      get :show, params: { id: workspace.id }, format: :pdf
-      expect(response).to be_successful
+  context 'GET /worspaces/:id/prepare_pdf' do
+    it 'should send workspace details to email' do
+      get :prepare_pdf, params: { id: workspace.id }, xhr: true
+      allow(WorkspaceDetailsJob).to receive(:perform_later).with(workspace.id, user.id)
```

Comment:
Where is an actual test? Allow only stubs a method it doesn't verify anything, write at least one expect()